_episodes/01-conda.md

```diff
@@ -26,19 +27,19 @@ which is the package manager associated with Anaconda
 
 ## Basic usage
 
-Anaconda comes with around 75 of the most widely used libraries (and their depedencies).
```

Comment:
These number are outdated and I think it is better not to mention then.

Reply:
Good pick up. It looks like the [latest numbers](https://docs.anaconda.com/anaconda/#anaconda-navigator-or-conda) are 150 come installed and there's an additional 250 available via `conda install`. I like to mention the numbers in the lesson just to give people an idea of the scale of what's available.

Reply:
OK